src/lib/types/move/pool.ts

```diff
@@ -0,0 +1,26 @@
+import type { HexAddr } from "../addrs";
+import type { Option } from "../common";
+import type { USD } from "../currency";
+
+export type MovePoolInfos = Record<
+  string,
+  {
+    coinA: {
+      metadata: HexAddr;
+      denom: string;
+      precision: number;
+      amountAPerShare: Big;
+      symbol: Option<string>;
+    };
+    coinB: {
+      metadata: HexAddr;
+      denom: string;
+      precision: number;
+      amountBPerShare: Big;
+      symbol: Option<string>;
+    };
+    lpPricePerShare: Option<USD<Big>>;
+    precision: number;
+    images: [Option<string>, Option<string>];
```

Comment:
What is the actual name we use for a pool's image? I've come across several terms like `images`, `logo`, and `poolLogo`.